### PR TITLE
change default port for sfstation

### PR DESCRIPTION
**Pull request by Crabzmatic**

### CODE REVIEW

**Review by Temppus — APPROVED** [review]

File: sf-docker/.env.sfstation [hunk 1]
```diff
@@ -2,7 +2,7 @@
 HOST=0.0.0.0
 
 # Port number, where SF AccessStation server will bind to
-PORT=80
+PORT=8000
```
Comment (innoqazmlp) [inline sf-docker/.env.sfstation:5]: Not sure if this is not overriden by sfstation service definition in docker-compose.yml file. Also port mapping is probably needed to update there

Comment (Crabzmatic) [reply sf-docker/.env.sfstation:5]: Like this? See commit 3b8820b

Comment (innoqazmlp) [reply sf-docker/.env.sfstation:5]: yes. Also it seems to me that environment section in docker-compose can be removed

Comment (Crabzmatic) [reply sf-docker/.env.sfstation:5]: done

**Review by innoqazmlp — APPROVED** [review]

**Pull request merged**